277 masvs threatmodel (#280)
OWASP/owasp-masvs: Document/CHANGELOG.md

This document is automatically generated at {{gitbook.time}}

## V1.2 - WIP - do not translate yet

The following changes are part of release 1.2:

> - Promoted to flagship status

Reviewer: ```suggestion
- Promoted to flagship status.
```

> - Requirement changed: MSTG-STORAGE-1 "need to be used"

Reviewer: ```suggestion
- Requirement changed: MSTG-STORAGE-1 "need to be used".
```

> - Requirement MSTG‑AUTH‑12, MSTG‑STORAGE‑13 & MSTG‑STORAGE‑14 are added with a focus on data protection.

Reviewer: Replaced with dashes

```suggestion
- Requirements MSTG-AUTH-12, MSTG-STORAGE-13 and MSTG-STORAGE-14 are added with a focus on data protection.
```

> - Requirement MSTG-RESILIENCE-13 is added as a defence in depth for device binding and/or anti-fraud information

Reviewer: Defense is the US term.

```suggestion
- Requirement MSTG-RESILIENCE-13 is added as a defense in depth for device binding and/or anti-fraud information.
```
